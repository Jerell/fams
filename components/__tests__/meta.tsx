import { mount, shallow, render } from 'enzyme';
import Meta from "../meta"

describe("<Meta/>", () => {
  it("should match snapshot", () => {
    const component = shallow(<Meta />)
    expect(component).toMatchSnapshot()
  })
})