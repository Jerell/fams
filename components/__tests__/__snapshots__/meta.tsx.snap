// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta/> should match snapshot 1`] = `
<Head>
  <title>
    Rice Bowl
  </title>
  <meta
    charSet="utf-8"
  />
  <meta
    content="yes"
    name="mobile-web-app-capable"
  />
  <meta
    content="yes"
    name="apple-mobile-web-app-capable"
  />
  <meta
    content="black-translucent"
    name="apple-mobile-web-app-status-bar-style"
  />
  <meta
    content="Rice Bowl"
    name="apple-mobile-web-app-title"
  />
  <meta
    content="Rice Bowl"
    name="application-name"
  />
  <meta
    content="Bring your own ingredients"
    name="description"
  />
  <meta
    content="#1d2020"
    name="theme-color"
  />
  <meta
    content="width=device-width, initial-scale=1, user-scalable=0, viewport-fit=cover"
    name="viewport"
  />
  <link
    href="/images/icon-maskable-512.png"
    rel="apple-touch-icon"
  />
  <link
    href="/images/favicon.png"
    rel="icon"
    type="image/png"
  />
  <link
    href="/manifest.json"
    rel="manifest"
  />
</Head>
`;
